cms/envs/labster.py

```diff
@@ -24,3 +24,10 @@
 RAVEN_CONFIG = AUTH_TOKENS.get('RAVEN_CONFIG', {})
 if RAVEN_CONFIG.get('dsn'):
     INSTALLED_APPS += ('raven.contrib.django.raven_compat',)
+
+# https://github.com/edx/edx-platform/wiki/Optional-Password-Policy-Enforcement
+PASSWORD_MIN_LENGTH = 8
```

Comment:
What are default values for these settings?

Reply:
print(settings.PASSWORD_MIN_LENGTH) = None
FYI @TamoshaytisV